### PR TITLE
[Imaging Browser][Issue Tracker] Please go away, back url

### PR DESCRIPTION
**Pull request by AnyhowStep**
In reference to Redmine 12316,

That back url is ugly. Made it so that just the mere existence of the `backURL` in the query param is enough to add a back button. Without it, it doesn't add it. With that, we can only go "back" to one URL, though. Pretty sure that's all we need in these cases.

_EDIT: (by @johnsaigle) this is also a security improvement._

### CODE REVIEW

**Comment by AnyhowStep** [comment]
Err, I didn't run `phpcs`. It will probably fail that. I'll fix that later.

**Comment by AnyhowStep** [comment]
Wow, it passed.

File: modules/issue_tracker/php/issue_tracker_ControlPanel.class.inc [hunk 1]
```diff
@@ -68,8 +68,10 @@ class Issue_Tracker_ControlPanel
     function getData()
     {
         $factory = NDB_Factory::singleton();
-        $issueData['backURL'] = $factory->settings()->getBaseURL()
-            . urldecode($_REQUEST['backURL']);
+        $issueData['backURL'] = isset($_REQUEST['backURL']) ?
+            $factory->settings()->getBaseURL()
+                . "/issue_tracker/" :
+            null;;
```
Comment (johnsaigle) [inline modules/issue_tracker/php/issue_tracker_ControlPanel.class.inc]: Two semicolons? 😂 👯‍♂️

**Comment by johnsaigle** [comment]
@AnyhowStep I issued a tiny little commit to fix your typo. Could you just look over this a final time?

**Comment by AnyhowStep** [comment]
Let me make a few more changes. Since the branch is now 17.1

**Comment by johnsaigle** [comment]
@anyhowstep 17.0 you mean? 😃

**Comment by AnyhowStep** [comment]
@driusan changed the base branch to aces:17.1-dev from aces:17.0-dev 21 hours ago =(

**Comment by johnsaigle** [comment]
Yeah PRs related to security are always going to go to the lowest-numbered branch.

**Comment by johnsaigle** [comment]
@driusan Can you clarify whether this should be 17.1 or 17.0?

**Comment by AnyhowStep** [comment]
So.. Uh... Does this need a "discussion required" tag?

**Comment by johnsaigle** [comment]
Tested on both modules and the redirect works without issue.

**Pull request merged**